named_entity_recognition_bert/Legal-NER.ipynb

```diff
@@ -0,0 +1,1802 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4c569da6-7eb9-4a62-afef-8764aa2fce1a",
+   "metadata": {},
+   "source": [
+    "Now we can download labels from SuperAnnotate for the train texts that were annotated manually. The annotations will be downloaded in [SuperAnnotate format](https://doc.superannotate.com/docs/sdk-export-annotations)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "bff0e565",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "token = \"e7514d15a4e0a2bc3824e864bada3aa12183c2aff6ce8910cd14f719946cc6ba797b17ddb4d0b651t=19366\"\n",
+    " \n",
+    "sa_client = SAClient(token = token)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "e13e258f",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "SA-PYTHON-SDK - INFO - Getting 3620 annotations from Legal-NER/train.\n",
+      "Processing: 100%|██████████| 3620/3620 [00:15<00:00, 230.67it/s]\n"
+     ]
+    }
+   ],
+   "source": [
+    "sa_response = sa_client.get_annotations(project=\"Legal-NER/train\",\n",
+    "                                        items=[os.path.basename(x) for x \\\n",
+    "                                               in data_links_dict['train']])\n",
+    "\n",
+    "annotations = [i['instances'] for i in sa_response]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "c77d598e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "unique_labels = set([entity['className'] for a in annotations for entity in a])\n",
+    "unique_labels.add('O')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "id": "dd803984",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'B-CASE_NUMBER',\n",
+       " 'B-COURT',\n",
+       " 'B-DATE',\n",
+       " 'B-GPE',\n",
+       " 'B-JUDGE',\n",
+       " 'B-LAWYER',\n",
+       " 'B-ORG',\n",
+       " 'B-OTHER_PERSON',\n",
+       " 'B-PETITIONER',\n",
+       " 'B-PRECEDENT',\n",
+       " 'B-PROVISION',\n",
+       " 'B-RESPONDENT',\n",
+       " 'B-STATUTE',\n",
+       " 'B-WITNESS',\n",
+       " 'I-CASE_NUMBER',\n",
+       " 'I-COURT',\n",
+       " 'I-DATE',\n",
+       " 'I-GPE',\n",
+       " 'I-JUDGE',\n",
+       " 'I-LAWYER',\n",
+       " 'I-ORG',\n",
+       " 'I-OTHER_PERSON',\n",
+       " 'I-PETITIONER',\n",
+       " 'I-PRECEDENT',\n",
+       " 'I-PROVISION',\n",
+       " 'I-RESPONDENT',\n",
+       " 'I-STATUTE',\n",
+       " 'I-WITNESS',\n",
+       " 'O'}"
+      ]
+     },
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "unique_labels"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c763fb0d-44bb-48df-ba7c-b0648c8b3ea0",
+   "metadata": {},
+   "source": [
+    "We will map each label into its id and vice versa for the BERT model."
```

Comment:
What is vice versa? Are you Rome citizen? =)

Reply:
??